gate-web/src/main/groovy/com/netflix/spinnaker/gate/controllers/PagerDutyController.groovy

```diff
@@ -79,15 +80,23 @@ class PagerDutyController {
       log.info("Fetched {} PagerDuty services", services?.size())
       pagerDutyServicesCache.set(services)
     } catch (e) {
-      log.error("Unable to refresh PagerDuty service list", e)
+      if (e instanceof RetrofitError && e.response?.status == 429) {
+        log.warn("Unable to refresh PagerDuty service list (throttled!)")
```

Comment:
❤️

Reply:
at 5m intervals we actually get throttled? Should we add a "sleep/delay" if we are throttled so as not exacerbate the situation further?

Reply:
or actually look at the header and see what the delay value is? maybe not super important. Ideally, i guess these are on some sort of "agent" and globally cached, but def not worth it here